VADR Annotation Integration

ankushkgupta2 opened the pull request:
## Description
**Please Describe the Bug(s) Fixed and/or the Feature(s) Added:**
(1) VADR annotation is integrated into the Nextflow wrapper within the main workflow and as an entrypoint, running in parallel with other annotation options for submission. All proper parameters have been updated across all sets.
(2) Run_VADR.py script written to transfrom the tables outputted from VADR into properly formatted GFFs + producing other logging information for various checks.
(3) README.md is updated to reflect process, entrypoint, and parameters.
(4) Abstracted out general annotation related functions into a new script called annotation_utility.py for both Liftoff and VADR to share (as well as future annotation pieces).
(5) Needed mpxv-models for VADR annotation have been pushed to the repository and is available to others (needed to point the model dir ENV variable to these files external to the container when mounting).

## Checklist

**Go Through Checklist Below and Place A :heavy_check_mark: if Completed**

#### General Checks

* [X] Have you ran appropriate tests (unit/integration/end-to-end) to check logic across run environments (Conda/Docker/Singularity on Scicomp/AWS/NF Tower/Local)?

    
    For each relevant configuration:

    * Can the program run completely through without erroring out?
    * Does it produce the expected outputs, given the inputs provided? 

* [X] Have you conducted proper linting procedures?
    * Numpy formatted docstrings for functions
    * Comments explaining lines of code
    * Consistent and intuitive naming conventions for variables, functions, classes, methods, attributes, and scripts
    * Single empty line between class functions, two lines between non-class functions, and two lines between imports and code body
    * Camel case formatting for class names

* [X] Have you updated existing documentation (README.md, etc.) or created new ones within docs?

#### CDC Checks

* [X] Did you check for sensitive data, and remove any?
* [No] Are additional approvals needed for this change?
* [No] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?
* [N/A] If you added or modified HTML, did you check that it was 508 compliant?

kyleoconnell reviewed (changes requested):
LN 20: Change non-scicomp setup to non-CDC setup
LN 80: ### Environment Setup, carve out a section her for CDC users on sci-comp servers. All you need to do then is run it with `ml nextflow` and then profile conda/singularity,standard. 
LN 89: remove the gitlab url since it is internal. Only mention the MPOX gitlab repo, but not the path.
Can you make it more clear what inputs are needed for submission only?

ankushkgupta2 commented:
> LN 20: Change non-scicomp setup to non-CDC setup LN 80: ### Environment Setup, carve out a section her for CDC users on sci-comp servers. All you need to do then is run it with `ml nextflow` and then profile conda/singularity,standard. LN 89: remove the gitlab url since it is internal. Only mention the MPOX gitlab repo, but not the path. Can you make it more clear what inputs are needed for submission only?

Addressed all concerns above and re-requested a review.
